core/templates/dev/head/components/OutcomeDestinationEditorDirective.js

```diff
@@ -22,17 +22,18 @@ oppia.directive('outcomeDestinationEditor', [
       restrict: 'E',
       scope: {
         outcomeHasFeedback: '=',
-        outcome: '='
+        outcome: '=',
+        addState: '='
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/outcome_destination_editor_directive.html'),
       controller: [
-        '$scope', 'EditorStateService', 'ExplorationStatesService',
+        '$scope', 'EditorStateService',
```

Comment:
Perhaps this should be StateEditorService since it is "in service of" the state-editor directive.